### DISCUSSION TITLE
How to do genotype imputation with HRC panel without using online service

### DISCUSSION THREAD

**Discussion by xuz1** [genetics-of-dna-methylation-consortium/godmc_phase2#8] [Q&A]
We are not allowed to upload genotype data. Is there any way to download the HRC reference panel to do imputation on a local server?

**Comment by epzjlm** [comment]
The HRC [paper by McCarthy et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388176/) suggests that the full HRC is not publicly available:

"Since the HRC reference panel combines data from many different studies with a range of restrictions on data release we have developed centralized imputation server resources (see [URLs](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5388176/#FN3)). Under this model researchers upload phased or unphased genotype data and imputation is carried out on central servers
....
A limited subset of HRC haplotypes will be made available for researchers via the European Genome-phenome Archive (EGA) for the sole purpose of phasing and imputation."

In addition I am also not aware of plans to develop the imputation service outside UK and US.

**Comment by vera-karlbauer** [comment]
There's a new European imputation server (hosted in Munich, Germany) where you can impute using HRC: https://imputationserver.helmholtz-munich.de
This is still an online server, but if you can only share/upload genotype data within the EU, this might be a good option.

**Comment by epzjlm** [comment]
That is fantastic news!
